include/oneapi/dpl/pstl/hetero/numeric_impl_hetero.h

```diff
@@ -108,23 +118,216 @@ __pattern_transform_reduce(_ExecutionPolicy&& __exec, _ForwardIterator __first,
     return __res;
 }
 
+template< bool _Inclusive>
+struct __single_group_scan
+{
+    template<typename _Policy, typename _InRng, typename _OutRng, typename _InitType, typename _BinaryOperation, typename _UnaryOp>
+    static void __launch_dynamic_bounds_scan(_Policy const & __policy, _InRng __in, _OutRng __out, ::std::size_t __n, _InitType __init, const _BinaryOperation& __bin_op, const _UnaryOp& __unary_op, ::uint16_t __wg_size)
```

Comment:
inconsistent `const` position for a reference.

Reply:
Will fix.